requirements.txt

```diff
@@ -16,3 +16,8 @@ v3io>=0.3.3
 matplotlib
 scikit-learn
 seaborn
+azure-mgmt-resource
```

Comment:
@theSaarco are all those needed ? always or some only in dev?

Reply:
Actually, I think that's a good catch and these are leftovers from other pieces of code that I worked with. For our purposes (i.e. accessing storage blobs) the only one needed should be azure-storage-blob.
I'll remove the redundant lines.